docker-compose.yml

```diff
@@ -28,3 +28,8 @@ services:
       - "5000:5000"
     volumes: 
       - ./backend:/var/backend
+  mongo:
```

Comment:
to be consistent with the postgres service, i would call this service `mongo-db`